cmake/KokkosConfigCommon.cmake.in

```diff
@@ -9,6 +9,10 @@ SET(Kokkos_CXX_COMPILER_ID "@KOKKOS_CXX_COMPILER_ID@")
 FOREACH(DEV ${Kokkos_DEVICES})
   SET(Kokkos_ENABLE_${DEV} ON)
 ENDFOREACH()
+IF(Kokkos_ENABLE_THREADS)  # for backward compatibility
+  SET(Kokkos_ENABLE_PTHREAD ON)
+  LIST(APPEND Kokkos_DEVICES PTHREAD)
+ENDIF()
 
 IF(NOT Kokkos_FIND_QUIETLY)
   MESSAGE(STATUS "Enabled Kokkos devices: ${Kokkos_DEVICES}")
```

Comment:
So we print `Kokkos_THREADS` and `Kokkos_PTHREAD` after this pull request? Is omitting to add `PTHREAD` to `Kokkos_DEVICES` feasible?

Reply:
I will append after printing.  Unfortunately we have to append `PTHREAD` to `Kokkos_DEVICES` and set `Kokkos_ENABLE_PTHREAD` to `ON` or we may break code downstream.

Reply:
Now only appending if Kokkos_ENABLE_DEPRECATED_CODE_3 was ON

Reply:
Looks good.